docs/static/releasenotes.asciidoc

```diff
@@ -11,6 +12,49 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-1-1]]
+=== Logstash 6.1.1 Release Notes
+*  There are no user-facing changes in Logstash core in this release.
+
+[float]
+==== Input Plugins
+
+*`Beats`*:
+
+* Fix issue with close_wait connections, making sure keep alive sent back to the client. (https://github.com/logstash-plugins/logstash-input-beats/pull/272[#272])
+
```

Comment:
I'd tweak this slightly, either to say:

"Fix issue with close_wait connections to make sure that keep live is sent back to the client."